globals/card_definitions.gd

```diff
@@ -202,6 +202,8 @@ func get_condition_text(condition, amount, amount2, detail):
 			text += "If initiated strike, "
 		"hit_opponent":
 			text += "If hit opponent, "
+		"last_turn_was_strike":
+			text += "If last turn was a strike, "
```

Comment:
might also want one for not_last_turn_was_strike

Reply:
I don't think this actually comes up, but I'll add it anyway.